SECURITY.md

```diff
@@ -0,0 +1,28 @@
+# Security Policy
+
+## Supported Versions
+
+This project supports the following versions with security updates:
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 1.2.0   | :white_check_mark: |
+| 1.1.x   | :white_check_mark: |
+| 1.0.x   | :white_check_mark: |
```

Comment:
We do not provide any updates for these versions anymore. We just support the latest version, of today 1.2.X